plugins/github-actions/src/components/Cards/RecentWorkflowRunsCard.test.tsx

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import type { Props as RecentWorkflowRunsCardProps } from './RecentWorkflowRunsCard';
+import React from 'react';
+import { render } from '@testing-library/react';
+import { RecentWorkflowRunsCard } from './RecentWorkflowRunsCard';
+import { useApi } from '@backstage/core-api';
+import { useWorkflowRuns } from '../useWorkflowRuns';
+import { ThemeProvider } from '@material-ui/core';
+import { lightTheme } from '@backstage/theme';
+import { MemoryRouter } from 'react-router';
+
+jest.mock('../useWorkflowRuns', () => ({
+  useWorkflowRuns: jest.fn(),
+}));
+jest.mock('@backstage/core-api');
+
+describe('<RecentWorkflowRunsCard />', () => {
+  const entity = {
+    apiVersion: 'v1',
+    kind: 'Component',
+    metadata: {
+      name: 'software',
+      annotations: {
+        'github.com/project-slug': 'theorg/the-service',
+      },
+    },
+    spec: {
+      owner: 'guest',
+      type: 'service',
+      lifecycle: 'production',
+    },
+  };
+
+  const workflowRuns = [1, 2, 3, 4, 5].map(n => ({
+    id: `run-id-${n}`,
+    message: `Commit message for workflow ${n}`,
+    source: { branchName: `branch-${n}` },
+    status: 'completed',
+  }));
+  const mockErrorApi = { post: jest.fn() };
+
+  beforeEach(() => {
+    (useWorkflowRuns as jest.Mock).mockReturnValue([{ runs: workflowRuns }]);
+    (useApi as jest.Mock).mockReturnValue(mockErrorApi);
```

Comment:
should remove this mocking, and instead inject it in the component tree when rendering - see this example https://github.com/spotify/backstage/blob/8b53edb47489cb9e60cfc25f9981e4bb066852a1/plugins/register-component/src/components/RegisterComponentPage/RegisterComponentPage.test.tsx#L49

Reply:
done